e2e/test/scenarios/visualizations-tabular/pivot_tables.cy.spec.js

```diff
@@ -1272,6 +1272,51 @@ describe("scenarios > visualizations > pivot tables", { tags: "@slow" }, () => {
       cy.findByTestId("qb-save-button").should("have.attr", "data-disabled");
     });
   });
+
+  describe("issue 38265", () => {
+    beforeEach(() => {
+      createQuestion(
+        {
+          query: {
+            "source-table": ORDERS_ID,
+            aggregation: [
+              ["count"],
+              ["sum", ["field", ORDERS.SUBTOTAL, {"base-type": "type/Float"}]],
+            ],
+            breakout: [
+              ["field", ORDERS.CREATED_AT, {"base-type": "type/DateTime", "temporal-unit": "month"}],
+              ["field", PEOPLE.STATE, {"base-type": "type/Text", "source-field": ORDERS.USER_ID}],
+              ["field", PRODUCTS.CATEGORY, {"base-type": "type/Text", "source-field": ORDERS.PRODUCT_ID}],
+            ],
+          },
+          display: "pivot",
+        },
+        {
+          wrapId: true,
+          idAlias: "questionId",
+        },
+      );
+    });
+
+    it.only("correctly filters the query when zooming in on a **row** header (metabase#38265)", () => {
+      cy.signInAsNormalUser();
+      visitQuestion("@questionId");
+      cy.findByTestId("pivot-table").findByText("KS").click();
+      cy.wait(1);
+      popover().findByText("Zoom in").click();
+      cy.wait(1);
+
+      // Check for the filter pill
+      const filters = cy.findAllByTestId("filter-pill");
+      filters.should("have.length", 1);
+      filters.first().should("have.text", "User → State is KS");
+
+      // And the new pivot column headers
+      cy.findByTestId("pivot-table").findByText("Created At: Month").should("exist");
```

Comment:
`cy.findAllByTestId("pivot-table").should("contain", "Created At: Month").should("contain", ...`